### PR TITLE
containers widgets ui

### PR DESCRIPTION
**Pull request by tomaszrondio**
not perfect but should be cool enough for start

### CODE REVIEW

File: app/components/containerControls/containerControls.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ng-transclude></ng-transclude>
+
+<div class="blockControls swp_container__controls">
+    <a class="blockControls__item" >Container<i class="icon-pencil icon--white"></i></a>
+</div>
+
```
Comment (takeit) [inline app/components/containerControls/containerControls.html]: not needed new line

File: app/components/containerControls/containerControls.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    .swp_container{
+        outline: 1px dashed $colorContainer;
+        outline-offset: 5px;
+        position: relative;
+
+        &__controls{
+            display: block;
+        }
+    }
+}
```
Comment (takeit) [inline app/components/containerControls/containerControls.scss]: new line is missing

File: app/components/toolbar/toolbar.html [hunk 3]
```diff
@@ -1,15 +1,12 @@
...
-        Live Management
-    </div>
-    <a class="toolbar__close" ng-click="toggleToolbar()">
-        <i class="icon-close-thick"></i>
-    </a>
-
-    <a class="toolbar__open" ng-click="toggleToolbar()">
-    </a>
-
+<div class="leToolbar" ng-class="{'leToolbar--active': toolbarActive}">
```
Comment (takeit) [inline app/components/toolbar/toolbar.html:1]: the toolbar is transparent when there is more text and when you scroll down, see below:

zrzut ekranu 2017-01-25 15 58 35

File: app/components/widgetControls/widgetControls.html [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<ng-transclude></ng-transclude>
+
+<div class="blockControls swp_widget__controls">
+    <a class="blockControls__item blockControls__item--widget" >Widget Name<i class="icon-pencil icon--white"></i></a>
+</div>
+
```
Comment (takeit) [inline app/components/widgetControls/widgetControls.html]: new line not needed

File: app/components/widgetControls/widgetControls.scss [hunk 5]
```diff
@@ -0,0 +1,7 @@
+.leActive{
+    .swp_widget:hover{
+        outline: 1px dashed $colorWidget;
+        outline-offset: 5px;
+        position: relative;
+    }
+}
```
Comment (takeit) [inline app/components/widgetControls/widgetControls.scss]: new line is missing

File: app/style/_blockControls.scss [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        i:last-child{
+            margin-left: 1em;
+            font-size: 14px;
+        }
+    }
+}
+
+.swp_container .swp_widget:first-child .blockControls{
+    left: 89px;
+ }
```
Comment (takeit) [inline app/style/_blockControls.scss]: new line is missing

File: app/style/app.scss [hunk 7]
```diff
@@ -1,2 +1,9 @@
 @import '../../node_modules/superdesk-ui-framework/app/styles/_variables.scss';
+@import '_variables';
+@import '_breakpoints';
+@import '_helpers';
+
 @import './app/components/toolbar/toolbar.scss';
+@import './app/components/containerControls/containerControls.scss';
+@import './app/components/widgetControls/widgetControls.scss';
+@import '_blockControls';
```
Comment (takeit) [inline app/style/app.scss]: new line is missing

File: app/style/_variables.scss [hunk 8]
```diff
@@ -0,0 +1,2 @@
+$colorContainer: #b2ebf2;
+$colorWidget: #673ab7;
```
Comment (takeit) [inline app/style/_variables.scss]: new line is missing

File: app/style/_helpers.scss [hunk 9]
```diff
@@ -0,0 +1,4 @@
+.leActive{
+  padding-top: $nav-height;
+  transition: all .2s;
+}
```
Comment (takeit) [inline app/style/_helpers.scss]: new line is missing

File: app/style/_breakpoints.scss [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+@mixin lg(){
+    @media (min-width: #{$screen-lg}){
+        @content;
+    }
+}
+@mixin xlg(){
+    @media (min-width: 1800px){
+        @content;
+    }
+}
```
Comment (takeit) [inline app/style/_breakpoints.scss]: new line is missing

**Review by takeit — CHANGES_REQUESTED** [review]
It looks awesome so far! 👍

**Review by pavlovicnemanja — APPROVED** [review]

**Review by takeit — APPROVED** [review]

**Pull request merged**